articles/azure/azs-how-deploy-sql-template-powershell.md

```diff
@@ -146,47 +152,47 @@ The images used to create this deployment are:
+| SqlServerServiceAccountUserName | The SQL server service account name                                           |                                                                       | sqlservice                                                                           |
+| SqlServerOffer                | The SQL server version                                                        | {SQL2016SP1-WS2016, SQL2016SP2-WS2016} | SQL2016SP2-WS2016              |
+| SqlServerSku                | The name of the SQL Server SKU                                                        | {Enterprise, Standard, SQLDEV}              | Enterprise
+| SqlStorageAccountName           | The name of SQL server storage account                                        |                                                                       |[tolower(concat(take(uniqueString(resourceGroup().id),8),'sql'))]
+| SqlStorageAccountType           | The type of the SQL server storage account created                            | {Premium_LRS, Standard_LRS}                                         | Premium_LRS                                                                        |
 | SqlSubnet                       | The address range of the SQL subnet created in the new VNET                   |                                                                       | 10.0.1.0/26                                                                          |
-| SqlVMSize                       | The size of the SQL VMs Created                                               | {Standard\_A1, Standard\_A2, Standard\_A3}                            | Standard\_A3                                                                         |
-| StaticSubnet                    | The address range of the subnet static IPs are allocated from in the new VNET |                                                                       | 10.0.0.0/24                                                                          |
-| TemplatesBaseUrl                | Linked Templates base url                                                     |                                                                       | https<!---->://raw.githubusercontent.com/Azure/AzureStack-QuickStart-Templates/master/sq... |
+| SqlVMSize                       | The size of the SQL VMs created                                               | {Standard_DS2_v2, Standard_DS3_v2, Standard_DS4_v2, Standard_DS5_v2, Standard_DS11_v2, Standard_DS12_v2, Standard_DS13_v2}                            | Standard_DS2_v2                                                                         |
+| NumberOfSqlVMDisks                       | The number of data disks                                               | {1...32}                            | 2                             |
```

Comment:
The number of data disks for what?

Reply:
`The number of data disks for each SQL VM`